src/routes/landing.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { Button, StyleSheet, Text, View, Image } from 'react-native';
+import colors from '../assets/colors.js'
+import images from '../assets/images.js'
+
+class Landing extends React.Component {
+  render() {
+    return (
+      <View style={styles.landing}>
+        <View style={styles.vContainer}>
+          <View style={styles.hContainer}>
+            <Button
+              title='Go To Search!!'
+              onPress={() => this.props.navigation.navigate('Search')}
+            />
+          </View>
+        </View>
+        <View style={styles.vContainer}>
+          <View style={styles.hContainer}>
+            <Text style={styles.text}>Almost half of the food in the U.S. goes to waste - approximately 3,000 pounds per second.</Text>
+          </View>
+        </View>
+        <View style={styles.vContainer}>
+          <View style={styles.hContainer}>
+          <Image source={images.recycleSymbol}/>
+            <Text style={styles.text}>Find local recycleing center</Text>
+          </View>
+        </View>
+        <View style={styles.vContainer}>
+          <View style={styles.hContainer}>
+          <Image source={images.camera}/>
+            <Text style={styles.text}>Autodetect item</Text>
+          </View>
+        </View>
+        <View style={styles.vContainer}>
+          <View style={styles.bottomRow}>
+            <View style={styles.imageContainer}>
+              <Image source={images.magGlass}/>
+            </View>
+            <View style={styles.imageContainer}>
+              <Image source={images.camera}/>
+            </View>
+            <View style={styles.imageContainer}>
+              <Image source={images.recycleBook}/>
+            </View>
+            <View style={styles.imageContainer}>
+              <Image source={images.gear}/>
+            </View>
+          </View>
+        </View>
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  landing: {
+    flex: 1,
+    flexDirection: 'column',
+    alignItems: 'center',
+    justifyContent: 'center',
+    backgroundColor: colors.transparent,
+  },
+  vContainer: {
+    flex: .2,
```

Comment:
flex of .... `.2`? That seems odd. This might be a react native thing, because it would be very strange for web.

Reply:
That is how I interpreted the use of Flex. 1 = 100%. 2=20%